website/content/commands/peering/generate-token.mdx

```diff
@@ -0,0 +1,68 @@
+---
+layout: commands
+page_title: 'Commands: Peering Generate Token'
+---
+
+# Consul Peering Generate Token
+
+Command: `consul peering generate-token`
+
+Corresponding HTTP API Endpoint: [\[POST\] /v1/peering/token](/api-docs/peering#generate-a-peering-token)
+
+The `peering generate-token` generates a peering token.
```

Comment:
```suggestion
The `peering generate-token` generates a peering token. The token is base 64-encoded string containing the token details. 
```
Saw this bit about the encoding in the main docs and thought it made sense to put here, too.